lib/fluent/plugin/out_splunk_hec.rb

```diff
@@ -23,6 +23,9 @@ class SplunkHECOutput < ObjectBufferedOutput
     config_param :index_key, :string, default: nil
     config_param :remove_index_key, :bool, default: false
     config_param :sourcetype, :string, default: nil
```

Comment:
Could you add `:deprecated` to this parameter like this?

https://github.com/fluent/fluentd/blob/068a0045ef4109eb0de6a581fe91628ceb204b1b/lib/fluent/plugin/filter_grep.rb#L41

Reply:
Good idea, fixed it!